package.json

```diff
@@ -159,7 +159,7 @@
   },
   "dependencies": {
     "@csstools/css-parser-algorithms": "^2.4.0",
-    "@csstools/css-tokenizer": "^2.2.1",
+    "@csstools/css-tokenizer": "^2.2.2",
```

Comment:
see : https://github.com/dependabot/dependabot-core/issues/7646

Reply:
Thanks for the manual fix!